File: PRAKTIKUM 4/PRAK404_2310817220028_SheilaSabina.c
#include <stdio.h>

int main() {
    int angka;
    float angka1, angka2, hasil;

    do {
        printf("\nInput\n");
        printf("Pilih program\n");
        printf("1. Penjumlahan\n");
        printf("2. Pengurangan\n");
        printf("3. Perkalian\n");
        printf("4. Pembagian\n");
        printf("5. Exit\n");
        printf("Masukkan Pilihan :");
        scanf("%d", &angka);

        if (angka >= 1 && angka <= 4) {
            printf("Masukkan nilai pertama : ");
            scanf("%f", &angka1);
            printf("Masukkan nilai kedua : ");
            scanf("%f", &angka2);

            switch (angka) {
                case 1:
                    hasil = angka1 + angka2;
                    printf("Hasil Penjumlahan antara %.2f dengan %.2f adalah %.2f\n", angka1, angka2, hasil);
                    break;
                case 2:
                    hasil = angka1 - angka2;
                    printf("Hasil Pengurangan antara %.2f dengan %.2f adalah %.2f\n", angka1, angka2, hasil);
                    break;
                case 3:
                    hasil = angka1 * angka2;
                    printf("Hasil Perkalian antara %.2f dengan %.2f adalah %.2f\n", angka1, angka2, hasil);
                    break;
                case 4:
                    if (angka2 != 0) {
                        hasil = angka1 / angka2;
                        printf("Hasil Pembagian antara %.2f dengan %.2f adalah %.2f\n", angka1, angka2, hasil);
                    } else {
                        printf("Error! Pembagian dengan nol tidak diperbolehkan.\n");
                    }
                    break;
            }
        } else if (angka == 5) {
            printf("Terimakasih, telah menggunakan kalkulator Sheila Sabina\n");
        } else {
            printf("Input anda salah, silahkan coba lagi\n");
        }

    } while (angka != 5);

    return 0;
}

File: PRAKTIKUM 4/PRAK404_2310817220028_SheilaSabina.py
while True:
    print("\nInput")
    print("Pilih program")
    print("1. Penjumlahan")
    print("2. Pengurangan")
    print("3. Perkalian")
    print("4. Pembagian")
    print("5. Exit")
    angka = int(input("Masukkan Pilihan: "))

    if 1 <= angka <= 4:
        angka1 = float(input("Masukkan nilai pertama: "))
        angka2 = float(input("Masukkan nilai kedua: "))

        if angka == 1:
            hasil = angka1 + angka2
            print(f"Hasil Penjumlahan antara {angka1:.2f} dengan {angka2:.2f} adalah {hasil:.2f}")
        elif angka == 2:
            hasil = angka1 - angka2
            print(f"Hasil Pengurangan antara {angka1:.2f} dengan {angka2:.2f} adalah {hasil:.2f}")
        elif angka == 3:
            hasil = angka1 * angka2
            print(f"Hasil Perkalian antara {angka1:.2f} dengan {angka2:.2f} adalah {hasil:.2f}")
        elif angka == 4:
            if angka2 != 0:
                hasil = angka1 / angka2
                print(f"Hasil Pembagian antara {angka1:.2f} dengan {angka2:.2f} adalah {hasil:.2f}")
            else:
                print("Error! Pembagian dengan nol tidak diperbolehkan.")
    elif angka == 5:
        print("Terimakasih, telah menggunakan kalkulator Sheila Sabina")
        break
    else:
        print("Input Anda salah, silakan coba lagi")